src/app-init/routes.js

```diff
@@ -1,4 +1,8 @@
 export const ROUTE_CMS = '/cms';
 export const ROUTE_CMS_CONTENTMODEL_LIST = '/cms/content-models';
 export const ROUTE_CMS_CONTENTMODEL_ADD = '/cms/content-models/add';
+export const ROUTE_CMS_EDIT_CONTENT = '/cms/content/edit';
```

Comment:
the edit should also contain the placeholder for the ID of the content itself (see content model edit route)